gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -720,3 +719,23 @@ def expand(self, pcoll):
         coder=_ToVcfRecordCoder(),
         compression_type=self._compression_type,
         header=self._header)
+
+
+class _WriteVcfDataLinesFn(beam.DoFn):
+  """A function that writes variants to one VCF file."""
+
+  def __init__(self):
+    self._coder = _ToVcfRecordCoder()
+
+  def process(self, (file_path, variants), *args, **kwargs):
+    # type: (Tuple[str, List[Variant]]) -> None
+    with filesystems.FileSystems.create(file_path) as file_to_write:
+      for variant in variants:
+        file_to_write.write(self._coder.encode(variant))
+
+
+class WriteVcfDataLines(PTransform):
+  """A PTransform for writing VCF data lines."""
```

Comment:
Please add a brief note about how this is different from WriteToVcf transform above now that we're keeping both.